script/Arcueid/Day 13/13_00_ARC13_5.txt

```diff
@@ -604,22 +608,22 @@ If you could truly see death―――you wouldn't even be able to stand."
 -- Page 438, Offset 32929.
 -- 喩えるのなら、それは月世界。
 // group pr check: change the title from The Lunar World to The Moon's Surface
-You could liken it to the surface of the moon.
+You could liken it to the moon's surface.
 }
 [sha:69c2cce44921389377299e5ed44c3c59c9188e85]{
 -- Page 438, Offset 32930.
 -- なにもかもが死に絶えた荒野に似ている。
-A cold wasteland where everything is dead.
+Resembling a cold wasteland where all lays dead.
```

Comment:
Prefer original

Reply:
Hate starting so many sentences with "A" or "An", especially when its not needed.

Far too many sentences in here do that already.

Reply:
It just does not read well to me as is /shrug
This one at least isn't too close to other ^A lines

Reply:
The elision of 似てる is because it's implicit in the EN - by likening it to something, he's already in a comparative mode, and then in the second sentence is just clarifying what aspects are what make it similar to the surface of the moon.

Reply:
なにもかもが死に絶えた荒野 would be implicit of the same thing in JP!

Well, whatever. I can reverse, but it truly is with heavy heart.

Reply:
I also think it should be reverted. Starting lots of stuff with "A" or "An" is not a real problem, at least not compared to having it read fluently.